core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -182,6 +182,8 @@ angular.module('oppia').factory('ThreadDataService', [
           thread.status =
             action === ACTION_ACCEPT_SUGGESTION ? STATUS_FIXED : STATUS_IGNORED;
           _openThreadsCount -= 1;
+          thread.messages = _fetchMessages(threadId);
```

Comment:
This is going to be a costly operation right? Do we want to do this after each suggestion is accepted? Can we instead add the review message to a message cache (which I think exists). /cc @brianrodri

Reply:
Yes. please, I'd prefer adding the message to the cache as well. You should be able to look at the backend code (suggestion_services.resolve_suggestion is the function I think) to see the values you'll need to fill. Something like:

```javascript
thread.messages.push({
  message_id: thread.messages.length,
  ...
});
```

Right now we still use a raw JSON object for messages, so you may want to wait for my #8658 so you have a domain object to work with instead (not much will change in terms of work required, however).